### PR TITLE
start work on #379: support of place template

### LINKED ISSUE #379: [EN] Handle the 'place' template
- Wiktionary page: https://en.wiktionary.org/wiki/Iraq

Wikicode:
```
{{place|en|A country in the Middle East|modern=Republic of Iraq}}
```

Output:
```
<i>Place</i> <b>A country in the Middle East</b>
```

Expected:
```
A country in the Middle East; modern Republic of Iraq
```

---

Model link, if any: https://en.wiktionary.org/wiki/Template:place

### PR DESCRIPTION
**Pull request by lasconic**
- [ ] Fixes #379
- [x] Tests added/updated.
- [ ] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by lasconic** [comment]
Check.sh is not happy and I'm clueless for now.

File: scripts/get.py [hunk 1]
```diff
@@ -373,7 +373,29 @@ def strip_html(text: str) -> str:
             for a, subdef in zip("abcdefghijklmopqrstuvwxz", definition):
                 if isinstance(subdef, tuple):
                     for rn, subsubdef in zip(
-                        ["i", "ii", "iii", "iv", "v", "vi", "vii"], subdef
```
Comment (BoboTiG) [inline scripts/get.py:376]: Here you could do much more simple I think.
```python
from .user_functions import into_to_roman

for rn, subsubdef in enumerate(subdef, 1):
    print(f"{into_to_roman(rn).lower()}.".rjust(12), strip_html(subsubdef))
```

File: scripts/lang/en/places.py [hunk 2]
```diff
@@ -0,0 +1,2425 @@
+placetypes = {
```
Comment (BoboTiG) [inline scripts/lang/en/places.py:1]: Could you add a docstring as it is done for `langs.py`? And you can directly add the good Wiki page for the linking with your script.

**Comment by lasconic** [comment]
See https://github.com/BoboTiG/ebook-reader-dict/pull/390

**Pull request closed without merging**